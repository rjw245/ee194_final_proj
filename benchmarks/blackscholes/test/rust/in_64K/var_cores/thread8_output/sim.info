{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread8_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../rust/bs',
             '8',
             '../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12458.649997,
            6.18006,
            162460,
            0,
            0,
            0,
            484834,
            0,
            0,
            0,
            12472,
            0,
            0,
            0,
            301210,
            657536),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/thread8_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../rust/bs 8 ../../../inputs/in_64K.txt'],
 't_elapsed': 16671.443895101547,
 't_start': 1460954529.505751,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 4105064448L}
