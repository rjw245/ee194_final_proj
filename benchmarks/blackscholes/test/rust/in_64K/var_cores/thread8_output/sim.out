                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |  4219286006 |      231412 |      189016 |      185949 |      185780 |      382359 |      184966 |      185781
  Cycles                             |  2472599010 |  2472599010 |  2472599010 |  2472599010 |  2472599010 |  2472599010 |  2472599010 |  2472599010
  IPC                                |        1.71 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Time (ns)                          |   929548500 |   929548500 |   929548500 |   929548500 |   929548500 |   929548500 |   929548500 |   929548500
  Idle time (ns)                     |      348772 |   929463962 |   929486942 |   929488613 |   929489122 |   929428286 |   929491219 |   929487441
  Idle time (%)                      |        0.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |   147440279 |       23828 |       19691 |       19453 |       19514 |       41048 |       19412 |       19489
  num incorrect                      |     6707324 |        2497 |        2057 |        2042 |        1979 |        3591 |        1949 |        2001
  misprediction rate                 |       4.35% |       9.49% |       9.46% |       9.50% |       9.21% |       8.04% |       9.12% |       9.31%
  mpki                               |        1.59 |       10.79 |       10.88 |       10.98 |       10.65 |        9.39 |       10.54 |       10.77
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   858601668 |       29019 |       25094 |       24554 |       24593 |       49929 |       24444 |       24466
    num misses                       |         197 |         104 |         104 |         104 |          98 |         107 |          98 |         104
    miss rate                        |       0.00% |       0.36% |       0.41% |       0.42% |       0.40% |       0.21% |       0.40% |       0.43%
    mpki                             |        0.00 |        0.45 |        0.55 |        0.56 |        0.53 |        0.28 |        0.53 |        0.56
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |  2813918550 |       80038 |       65227 |       63212 |       62954 |      130226 |       62821 |       63140
    num misses                       |      162617 |         225 |         148 |         124 |         102 |         278 |         103 |         131
    miss rate                        |       0.01% |       0.28% |       0.23% |       0.20% |       0.16% |       0.21% |       0.16% |       0.21%
    mpki                             |        0.04 |        0.97 |        0.78 |        0.67 |        0.55 |        0.73 |        0.56 |        0.71
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |      162814 |         329 |         252 |         228 |         200 |         385 |         201 |         235
    num misses                       |         861 |         218 |         199 |         195 |         179 |         240 |         181 |         196
    miss rate                        |       0.53% |      66.26% |      78.97% |      85.53% |      89.50% |      62.34% |      90.05% |      83.40%
    mpki                             |        0.00 |        0.94 |        1.05 |        1.05 |        0.96 |        0.63 |        0.98 |        1.06
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |   858601668 |       29019 |       25094 |       24554 |       24593 |       49929 |       24444 |       24466
    num cache misses                 |    20496408 |        1455 |        1350 |        1353 |        1288 |        2239 |        1291 |        1349
    miss rate                        |       2.39% |       5.01% |       5.38% |       5.51% |       5.24% |       4.48% |       5.28% |       5.51%
    mpki                             |        4.86 |        6.29 |        7.14 |        7.28 |        6.93 |        5.86 |        6.98 |        7.26
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |  2813919341 |       80486 |       65674 |       63665 |       63446 |      131145 |       63267 |       63577
    num cache misses                 |      106364 |        2296 |        1850 |        1270 |        1148 |        2504 |        1157 |        1196
    miss rate                        |       0.00% |       2.85% |       2.82% |       1.99% |       1.81% |       1.91% |       1.83% |       1.88%
    mpki                             |        0.03 |        9.92 |        9.79 |        6.83 |        6.18 |        6.55 |        6.26 |        6.44
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |    20602967 |        3826 |        3278 |        2670 |        2503 |        4872 |        2545 |        2631
    num cache misses                 |       40025 |        3086 |        2764 |        2350 |        2206 |        3002 |        2212 |        2269
    miss rate                        |       0.19% |      80.66% |      84.32% |      88.01% |      88.13% |      61.62% |      86.92% |      86.24%
    mpki                             |        0.01 |       13.34 |       14.62 |       12.64 |       11.87 |        7.85 |       11.96 |       12.21
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |       40237 |        3162 |        2842 |        2397 |        2275 |        3132 |        2310 |        2356
    num cache misses                 |       34184 |         948 |         753 |         654 |        1036 |        1932 |         615 |        1020
    miss rate                        |      84.96% |      29.98% |      26.50% |      27.28% |      45.54% |      61.69% |      26.62% |      43.29%
    mpki                             |        0.01 |        4.10 |        3.98 |        3.52 |        5.58 |        5.05 |        3.32 |        5.49
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |       19008 |           0 |           0 |           0 |       19049 |           0 |           0 |           0
  average dram access latency (ns)   |       60.51 |         inf |         inf |         inf |       60.16 |         inf |         inf |         inf
  average dram queueing delay        |        7.08 |        0.00 |        0.00 |        0.00 |        6.73 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |        3927 |         119 |          31 |          45 |          29 |          83 |          33 |          49
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |          57 |          81 |         151 |         123 |         172 |         379 |         125 |         210
